src/main/java/uk/gov/ons/census/casesvc/model/entity/Case.java

```diff
@@ -1,20 +1,85 @@
 package uk.gov.ons.census.casesvc.model.entity;
 
+import java.util.List;
 import java.util.UUID;
 import javax.persistence.Column;
 import javax.persistence.Entity;
+import javax.persistence.EnumType;
 import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.SequenceGenerator;
 import javax.persistence.Table;
 import lombok.Data;
 
 @Data
 @Entity
 @Table(name = "cases")
 public class Case {
-  @Id private UUID id;
 
-  @Column private String stuff;
+  @Id
+  @SequenceGenerator(name = "caseRefGenerator", initialValue = 10000000)
+  @GeneratedValue(generator = "caseRefGenerator", strategy = GenerationType.SEQUENCE)
+  private Long caseRef;
```

Comment:
strictly, I suppose we should make this `long` instead of `Long` because it's not nullable